api/src/mongodb.ts

```diff
@@ -8,15 +8,15 @@ function buildConnection(app: Application) {
   return connection
 }
 
-var db_status = false;
+let db_status = false;
 
 export async function dbStatus(app: Application) {
   if (!db_status) { //if initial connection failed return 503
     return 503
   }
   const connection = buildConnection(app);
-  const client = new MongoClient(connection, { useUnifiedTopology: true, serverSelectionTimeoutMS: 1000 });
-  var db_code = 200;
+  const client = new MongoClient(connection, { serverSelectionTimeoutMS: 1000 });
```

Comment:
Is `useUnifiedTopology: true` the default now?  This is part of a change made to fix some startup issues we were having with the issuer kit api.

Reply:
Yes, this is the default in the new(er) driver interface: https://www.mongodb.com/docs/drivers/node/current/whats-new/#mongoclientoptions-interface